schemas/v1/8_miner_sector_events_idx.go

```diff
@@ -0,0 +1,14 @@
+package v1
+
+func init() {
+	patches.Register(
+		8,
+		`
+------------------------------------------------------------------
+-- Name: miner_sector_events_event_idx
+-- Model: miner.MinerSectorEvent
+-- ----------------------------------------------------------------
+CREATE INDEX miner_sector_events_event_idx ON {{ .SchemaName | default "public"}}.miner_sector_events USING btree(event) INCLUDE(height, miner_id);
```

Comment:
Suggest using `CONCURRENTLY` otherwise this is a blocking action that will lock the table